Monika After Story/game/script-topics.rpy

```diff
@@ -4560,6 +4560,57 @@ label monika_ageinf:
     m "But I don't believe that my life can be summed up in a number."
     m 5a "So I'll always love you for all eternity, [player]. Just remember that."
     return
+    
+    init 5 python:
+    for key in ["orchestra", "instruments", "band"]:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_orchestra')
+    monika_random_topics.append('monika_orchestra')
+
+label monika_orchestra:
+
+ m 2d "Hey [player], do you like orchestras?"
+ m 1a "I love the way that so many different instruments can get together and create such wonderous music."
+ m "It must take determination and passion to practice countless of hours for just a few songs..."
+ m "Playing piano is the closest I can get to experiencing music firsthand."
+ m 1f "That reminds me, I still haven't gotten around finishing that song I was writing for you..."
+ m 1g "...Well, that's a shame, I really wanted you to hear it!"
+
+#First encounter with topic:
+ m 1d "What about you, [player]? Do you play any instruments?"
+ menu: 
+
+#Choice 1 (Yes)
+  "Yes.":
+   $persistent.instrument = True
+   m 1b "Really? Which instrument do you play?"
+   $ done = False
+    
+   while not done:
```

Comment:
This loop is an infinite loop. You need something to either set `done = True` in the loop to quit or just not use a loop at all.

In your case, I don't think you need a loop unless you're trying to validate input, which is not what you appear to be doing.